modules/seqexec/server/src/main/scala/seqexec/server/ghost/GHOST.scala

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server.ghost
+
+import cats.data.Reader
+import cats.data.EitherT
+import cats.effect.Sync
+import cats.implicits._
+import edu.gemini.spModel.config2.Config
+import edu.gemini.spModel.seqcomp.SeqConfigNames._
+import edu.gemini.spModel.gemini.ghost.Ghost
+import gem.math.{Angle, HourAngle}
+import scala.concurrent.duration._
+import seqexec.model.dhs.ImageFileId
+import seqexec.model.Model.{Instrument, Resource}
+import seqexec.server.ConfigUtilOps._
+import seqexec.server._
+import seqexec.server.keywords.{GDSClient, GDSInstrument}
+import seqexec.server.ghost.GHOSTController._
+import squants.time.{Seconds, Time}
+
+final case class GHOST[F[_]: Sync](controller: GHOSTController[F])
+    extends InstrumentSystem[F]
+    with GDSInstrument {
+  override val gdsClient: GDSClient = controller.gdsClient
+
+  override val resource: Resource = Instrument.GHOST
+
+  override val sfName: String = "GHOST"
+
+  override val contributorName: String = "ghost"
+
+  override val observeControl: InstrumentSystem.ObserveControl =
+    InstrumentSystem.Uncontrollable
```

Comment:
Another teaching moment for me.  What is this?

Reply:
The naming is probably scary, it is used to indicate if you can interrupt observations